src/Module/Conversation/Network.php

```diff
@@ -283,7 +283,7 @@ protected static function parseRequest(array $parameters, array $get)
 
 		self::$forumContactId = $parameters['contact_id'] ?? 0;
 
-		self::$selectedTab = Session::get('network-tab', '');
+		self::$selectedTab = Session::get('network-tab', DI::pConfig()->get(local_user(), 'network.view', 'selected_tab', ''));
```

Comment:
The legacy config value is an array with each individual tab information, you can't use it directly in place of the new session variable.

Reply:
I just reintroduced the config variable that you introduced and that I replaced with a session variable: https://github.com/friendica/friendica/commit/4d15cc01e28a3a76f92cdb686d1766bd4ddc29fd#diff-40fbeb4a2eede127cdeaa40f6acfbc266f7ae4c304f5baf1b53183c8d21b2fd1R283

Reply:
Ah, never mind.